cmd/apps/vpn-client/vpn-client.go

```diff
@@ -70,7 +71,29 @@ func main() {
 		}
 	}
 
-	appClient := app.NewClient(nil)
+	var directIPsCh = make(chan net.IP, 100)
+
+	eventSub := appevent.NewSubscriber()
+
+	eventSub.OnTCPDial(func(data appevent.TCPDialData) {
+		log.WithField("event_type", data.Type()).
+			WithField("event_data", data).
+			Info("GOT EVENT")
```

Comment:
What do you think about changing the case of this message to the sentence case?

Reply:
@nkryuchkov that's debug log, just missed that, will remove